README.md

```diff
@@ -139,19 +141,131 @@ fuel-core run \
 --service-name fuel-ignition-node \
 --keypair {KEYGEN_SECRET_KEY} \
 --relayer {ETHEREUM_RPC_ENDPOINT} \
---ip=0.0.0.0 --port 4000 --peering-port 30333 \
+--ip=0.0.0.0 \
+--port 4000 \
+--peering-port 30333 \
+--db-path ~/.fuel-ignition \
+--snapshot {PATH_TO_CHAIN_CONFIGURATION_FOLDER}/ignition \
+--utxo-validation \
+--poa-instant false \
+--enable-p2p \
+--bootstrap-nodes /dnsaddr/mainnet.fuel.network \
+--sync-header-batch-size 100 \
+--relayer-v2-listening-contracts 0xAEB0c00D0125A8a788956ade4f4F12Ead9f65DDf \
+--relayer-da-deploy-height 20620434 \
+--relayer-log-page-size 2000 \
+--sync-block-stream-buffer-size 30 \
+--min-gas-price 1000 \
+--starting-gas-price 1000 \
+--gas-price-change-percent 10 \
+--gas-price-threshold-percent 50
+```
+All the values used for the L1 relayer are based on Ethereum Mainnet make sure `{ETHEREUM_RPC_ENDPOINT}` is a RPC communicating to Ethereum Mainnet network.
+
+Instead of directly placing your personal values on the command we advise you to use, for example, environment variables.
+
+### From source
+
+Clone the chain configuration folder :
+```
+git clone https://github.com/FuelLabs/chain-configuration
+```
+
+Clone the `fuel-core` repository : 
+```
+git clone https://github.com/FuelLabs/fuel-core.git
+```
+
+Go to the latest release tag for ignition on the `fuel-core` repository :
+```
+git checkout v0.40.2
+```
+
+Generate a keypair for your node :
+```
+cargo run --bin fuel-core-keygen -- new --key-type peering
+```
+and copy the secret key displayed.
+
+Run your node (change all variable with {} to your own personal variables):
+```bash
+cargo run --release --bin fuel-core -- run \
```

Comment:
You are missing "production" feature. The command should looks like `cargo build --release -p fuel-core-bin --no-default-features --features production`.

Reply:
Done